src/crawlee/crawlers/_abstract_http/_abstract_http_parser.py

```diff
@@ -30,6 +30,29 @@ async def parse(self, response: HttpResponse) -> TParseResult:
             Parsed HTTP response.
         """
 
+    @abstractmethod
+    async def parse_text(self, text: str) -> TParseResult:
+        """Parse text.
+
```

Comment:
could you add something meaningful? like what the text is supposed to be for example